src/plugins/plugin.gridlines.js

```diff
@@ -1,29 +1,41 @@
 'use strict';
 
-module.exports = function(Chart) {
-	var helpers = Chart.helpers;
-	var globalDefaults = Chart.defaults.global;
+var defaults = require('../core/core.defaults');
+var helpers = require('../helpers/index');
+var MODEL_KEY = '$gridlines';
 
-	// Stores all gridLines that should be displayed
-	var lines = [];
+// This is copied from core.scale.js, which is also required here and im not sure where it should be placed for both of them to access it
```

Comment:
This should not be here, but i dont know where to move it for both core.scale.js and this plugin to access it